packages/wallet-connect-sdk-core/src/index.ts

```diff
@@ -96,7 +301,18 @@ export class WcSdk {
         await WcSdk.disconnect(this.wcClient, this.session)
     }
 
-    async sendRequest(request: RequestArguments) {
+    /**
+     * Sends a request to the Wallet and it will call the RpcServer

```

Comment:
```suggestion
     * Sends a request to the Wallet, which will call the RpcServer

```
_not entirely sure here:_
**a.** The method sends a request to the wallet and THE METHOD calls the RPC server?
OR
**b.** The method sends a request to the wallet and THE WALLET calls the RPC server?

The suggestion covers case **b.**